client/src/containers/CategoryForm.js

```diff
@@ -1,21 +1,43 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
+import * as actions from "../actions";
```

Comment:
i would feel this my self preference but i prefer to do destructing here to get the specific action that i want, but you don't have to change this i think its more self preference thing